RUNNING_A_PROJECT.md

```diff
@@ -7,9 +7,16 @@ As a project manager you are expected to be available in the Slack channel for y
 
 We require the following:
 
+* We adhere to an *agile methodology* that includes these components:
+  - **There is a project story/tracking board** (see note below) that is maintained by the product owner (if there is one for the project) or the project manager/lead otherwise.  This provides a focal point for sprint planning and status tracking.
+  - **Scrums (aka standups) take place on a regular schedule** - every weekday, 3 times a week, etc. - whatever meets the needs of the particular project.  These are time-boxed (15 minutes) and follow the usual routine (What have you done since last scrum?, What will you be working on?, Do you have any blockers?).  It is normal for topics to be brought up during the scrum, which are placed in the "parking lot" to be discussed after the scrum concludes.  Also, the scrums serve as a good launching point for pair programming activity.
+  - **Development takes place in sprints**, the duration of which are particular to each project, but are typically one or two weeks in length.
```

Comment:
Well, we could adjust the language to make sprints optional but strongly encouraged.  I think they're valuable in so many ways, including establishing a cadence to project work and a sense of having a goal(s) for sprint delivery.

I do agree that it is sometimes tough to actually *hold* a sprint meeting, given the circumstances.

I do feel that, overall, we should establish a minimum set of standards/practices that a project lead must try to follow for an AV project.  Otherwise, we could have chaos reign and AV people could get a very wrong sense of how "agile" is supposed to work.  One of the old knocks on agile is that it is not disciplined - but, of course, it is very disciplined and that is one reason that it works.